### ISSUE TITLE
Problem with Unbalanced calls to begin/end appearance transitions for <UINavigationComtroller>

### ISSUE THREAD

**Issue by deafjam** [marcosgriselli/SwipeableTabBarController#27]
Hi, I'm trying to use SwipeableTabBarController, but if swipe fast, the screen is gets stuck or blank screen. Helps only restart APP.

**Comment by deafjam** [comment]
In my app, is 4 NavigationControllers with TableViews inside each one.

**Comment by marcosgriselli** [comment]
Hi @deafjam, what swipe animation are you using? Is this happening on a device or simulator?

**Comment by deafjam** [comment]
It's happens on device

**Comment by deafjam** [comment]
I can add you in TestFlight if you want

**Comment by marcosgriselli** [comment]
Testflight won't be useful. Is there any chance you can replicate the issue on a demo repository here and invite me?

**Comment by deafjam** [comment]
Maybe I can send you a project? My email deafjam@rambler.ru

**Comment by marcosgriselli** [comment]
I rather check it here. You can make the repo private and invite me.

**Comment by deafjam** [comment]
Sorry but private repo is paid for me.(

**Comment by marcosgriselli** [comment]
Try this on your `UIViewControllers` that have `UITableViews`: 

```swift
 override func scrollViewWillBeginDragging(_ scrollView: UIScrollView) {
        setTabBarSwipe(enabled: false)
    }
    
    override func scrollViewWillEndDragging(_ scrollView: UIScrollView, withVelocity velocity: CGPoint, targetContentOffset: UnsafeMutablePointer<CGPoint>) {
        setTabBarSwipe(enabled: true)
    }
```

**Comment by deafjam** [comment]
Ok, thank you, I'll try it

**Comment by deafjam** [comment]
it's better, but there's still a error and freeze. In the debugger "Unbalanced calls to begin / end appearance transitions for <UINavigationController: 0x10201ee00>"

**Comment by marcosgriselli** [comment]
@deafjam I just released version [1.0.10](https://github.com/marcosgriselli/SwipeableTabBarController/releases/tag/1.0.10) which solves this issue.
